aspnetcore/host-and-deploy/linux-apache.md

```diff
@@ -381,26 +384,34 @@ Modify the *helloapp.conf* file to enable URL rewriting and secure communication
 
 <VirtualHost *:80>
     RewriteEngine On
-    RewriteCond %{HTTPS} !=on
-    RewriteRule ^/?(.*) https://%{SERVER_NAME}/$1 [R,L]
+    RewriteCond   %{HTTPS} !=on
+    RewriteRule   ^(.*)$ https://%{HTTP_HOST}$1 [R=301,L]
 </VirtualHost>
 
 <VirtualHost *:443>
-    ProxyPreserveHost On
-    ProxyPass / http://127.0.0.1:5000/
-    ProxyPassReverse / http://127.0.0.1:5000/
-    ErrorLog /var/log/httpd/helloapp-error.log
-    CustomLog /var/log/httpd/helloapp-access.log common
-    SSLEngine on
-    SSLProtocol all -SSLv2
-    SSLCipherSuite ALL:!ADH:!EXPORT:!SSLv2:!RC4+RSA:+HIGH:+MEDIUM:!LOW:!RC4
-    SSLCertificateFile /etc/pki/tls/certs/localhost.crt
+    Protocols             h2 http/1.1
+    ProxyPreserveHost     On
+    ProxyPass             / http://127.0.0.1:5000/
+    ProxyPassReverse      / http://127.0.0.1:5000/
+    ErrorLog              /var/log/httpd/helloapp-error.log
+    CustomLog             /var/log/httpd/helloapp-access.log common
+    Header                always set Strict-Transport-Security "max-age=63072000;includeSubdomains;preload"
+    SSLEngine             on
+    SSLProtocol           all -SSLv3 -TLSv1 -TLSv1.1 -TLSv1.2
+    SSLHonorCipherOrder   off
+    SSLCompression        off
+    SSLSessionTickets     off
+    SSLUseStapling        On
+    SSLStaplingCache      "shmcb:logs/ssl_stapling(32768)"
+    SSLCertificateFile    /etc/pki/tls/certs/localhost.crt
```

Comment:
Doesn't your cert have to support OCSP stapling? I'd be vary of suggesting of everyone turning it on.

Reply:
Yes, so it's not something that should be generally enabled.

Reply:
This was suggested by the mozilla config tool, and it might be a 1.3 thing (i.e., does 1.3 support it by default? idk ... I'll check). I'll study and work with this further on Tuesday morning.